mitodl/ocw-data-parser
# Add unit tests

#### What are the relevant tickets?

https://github.com/mitodl/ocw-data-parser/issues/18

#### What's this PR do?

This adds unit tests to the project with a coverage level of 93% along with a codecov configuration so coverage reports are published.

#### How should this be manually tested?

Test requirements can be installed with `sudo pip install -r test-requirements.txt` and then the tests can be run with `python -m pytest --cov --cov-report xml`  All tests should pass and a coverage report should be generated.
